src/content/platform-integration/ios/app-extensions.md

```diff
@@ -241,8 +243,26 @@ use an iOS simulator to test your extension in debug mode.
             view.addSubview(flutterViewController.view)
             flutterViewController.view.frame = view.bounds
         }
+
+        override func viewDidDisappear(_ animated: Bool) {
+            super.viewDidDisappear(animated)
+            self.extensionContext?.cancelRequest(
+                withError: NSError(domain: Bundle.main.bundleIdentifier!, code: 0))
+        }
     }
     ```
+:::note
+1. If you do not cancel the request in `viewDidDisappear`, the `ShareViewController` will not be deallocated after being dismissed. 
+
+2. You have to pass in the bundle identifier as the error domain, otherwise it will result in a `EXC_BAD_INSTRUCTION` crash. 
+:::
+
+## Advanced scrolling behavior
+
+By default, flutter view does not receive any touch event in a share extension. 
+To support a scrollable list in the share extension, follow [this instruction][]. 
+
+[this instruction]: {{site.github}}/flutter/website/issues/164670#issuecomment-2740702986
```

Comment:
I simply [linked the instruction](https://github.com/flutter/flutter/issues/164670#issuecomment-2761952022) instead of copying it, since it's not a very common usage, and I wanted to keep the instruction short. But I'm ok either way.

Reply:
No, we don't want to link to instructions in a bug. Eventually the bug will be closed. Please put the information here. (I'd also like to edit it.)

Reply:
Oh I didn't do this yet, but we intend to simplify the API, so it's fine - let's keep it as it is for now.